h2o-algos/src/main/java/hex/tree/DTree.java

```diff
@@ -140,17 +140,16 @@ StringBuilder printLine(StringBuilder sb ) {
   // Records a column, a bin to split at within the column, and the MSE.
   public static class Split extends Iced {
     final public int _col, _bin;// Column to split, bin where being split
-    final DHistogram.NASplitDir _nasplit;
```

Comment:
why not final anymore?

Reply:
Because I initialize Split before validation, and during validation sometimes direction is changed.